src/views/ModelPlan/Documents/__snapshots__/index.test.tsx.snap

```diff
@@ -110,24 +110,306 @@ exports[`Model Plan Documents page matches snapshot 1`] = `
           Add a document
         </a>
         <div
-          class="margin-y-10"
-          data-testid="page-loading"
+          class=""
```

Comment:
nit: I recognize this is a snapshot, but found it odd that there is a empty classname